img_proof/scripts/cli.py

```diff
@@ -362,6 +362,12 @@ def main(context, no_color):
     help='The architecture of the compute image being tested. '
          'Default: x86_64.',
 )
+@click.option(
+    '--exclude',
+    help='Comma-separated exclusion list for tests by test names. Tests that match any part of any expressions will be skipped',
```

Comment:
This help message does not match the implementation on line 338. The match obtained via `expand_test_files` is an exact match not a fuzzy/wildcard match as indicated in the help message. For example based on the comment one would expect that `--exclude motd` would skip the `test_sles_motd.py` and `test_sles_sap_motd.py` tests but it does not, rathre it will generate an exception

```
import sys
sys.path.append('/work/rjschwei/devel/img-proof/img-proof')
import ipa_utils
ipa_utils.expand_test_files(['/work/rjschwei/devel/img-proof/usr'], ['motd'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/work/rjschwei/devel/img-proof/img_proof/ipa_utils.py", line 146, in expand_test_files
    return parse_sync_points(expanded_names, tests)
  File "/work/rjschwei/devel/img-proof/img_proof/ipa_utils.py", line 457, in parse_sync_points
    test_files.append(find_test_file(name, tests))
  File "/work/rjschwei/devel/img-proof/img_proof/ipa_utils.py", line 182, in find_test_file
    'Test file with name: %s cannot be found.' % test_name
img_proof.ipa_exceptions.IpaUtilsException: Test file with name: motd cannot be found.
ipa_utils.expand_test_files(['/work/rjschwei/devel/img-proof/usr'], ['test_sles_motd'])
['/work/rjschwei/devel/img-proof/usr/share/lib/img_proof/tests/SLES/test_sles_motd.py']
```
Test expansion works on exact name matches  not on "match any part of any expressions"

Reply:
Good catch, I forgot to update the help message according to the latest changes and updated it now accordingly.

Please check and resolve if you agree with the current state.